library/Faktory/Worker.hs

```diff
@@ -90,8 +91,9 @@ processorLoop client settings workerSettings f = do
   let
     namespace = connectionInfoNamespace $ settingsConnection settings
     processAndAck job = do
-      f job
-      ackJob client job
+      void . timeout (getJobTTL job) $ do
```

Comment:
> The current implementation is silent, no information is logged, the job is just terminated. That is likely not desirable.

Originally I thought throwing a `JobReservationExpired` would make sense, but I now realize that would crash the main loop, which we don't want. Given that, I'd probably vote for `settingsLogError`.

Reply:
That was my thought as well. Crashing the consumer isn't great, but being invisible is also not great. Logging is a nice middle ground.